tests/ci/integration/run_trousers_integration.sh

```diff
@@ -33,10 +33,11 @@ rm -rf "${SCRATCH_FOLDER:?}"/*
 pushd "${SCRATCH_FOLDER}"
 
 function trousers_build() {
+  git apply "${SCRIPT_DIR}/trousers_patch/aws-lc-trousers.patch"
   sh ./bootstrap.sh
-  ./configure --with-gui=none --prefix="${TROUSERS_INSTALL_FOLDER}" --with-openssl="${AWS_LC_INSTALL_FOLDER}"
+  CFLAGS="-Werror=implicit-function-declaration -Wno-deprecated" ./configure --with-gui=none --prefix="${TROUSERS_INSTALL_FOLDER}" --with-openssl="${AWS_LC_INSTALL_FOLDER}"
```

Comment:
does the configure script detect `CFLAGS`? i would've thought this should be exported or passed in the `make` invocation....